command/acl/token/formatter_test.go

```diff
@@ -155,6 +155,7 @@ func TestFormatTokenList(t *testing.T) {
 			tokens: []*api.ACLTokenListEntry{
 				{
 					AccessorID:  "fbd2447f-7479-4329-ad13-b021d74f86ba",
+					SecretID:    "257ade69-748c-4022-bafd-76d27d9143f8",
```

Comment:
I think the command/acl/token/formatter.go also needs updating to output the SecretIDs if they are not redacted. Then the golden files for the `pretty` format should also be updated.

Reply:
I added the `SecretID` to the formatter. The only caveat is that the `SecretID` field is always shown rather than being omitted when it is redacted. This matches the API's behavior but wanted to make sure this is what we want for the CLI too vs only showing it when it's not redacted (as you mentioned above).

Reply:
Does that mean that it will show the secret ID as `<hidden>`? If so that seems fine. There probably isn't a good reason to hide the existence of the field all together in that case.